node/errors.js

```diff
@@ -259,6 +259,11 @@ module.exports.TChannelCallResContBeforeInitResError = TypedError({
     message: 'call response cont before init response'
 });
 
+module.exports.TChannelConnectionResetError = WrappedError({
+    type: 'tchannel.connection.reset',
+    message: 'tchannel: {origMessage}'
```

Comment:
Recommend cause and causeMessage properties. I contributed a change to the errors module to make these property names more concise.